src/main/resources/static/index.html

```diff
@@ -30,52 +30,54 @@
 			<input type="text" class="text-input" id="search-bar" />
 			<i class="material-icons">search</i>
 		</div>
-		<div class="section card">
-			<div class="section title">
-				<span>Hot</span>
-			</div>
-			<div class="section perk-list" id="top-perks">
-				<div class="loader"></div>
-				<!--
-				<div class="section perk-tile">
-					<div class="perk-vote">
-						<div class="perk-upvote perk-vote-button" onclick=vote(this,1)>
-							<i class="material-icons">thumb_up</i>
-						</div>
-						<div class="perk-score">
-							5
-						</div>
-						<div class="perk-downvote perk-vote-button" onclick=vote(this,-1)>
-							<i class="material-icons">thumb_down</i>
-						</div>
-					</div>
-					<div class="perk-info">
-						<div class="perk-name subtitle">
-							<span>10% off Starbucks</span>
-						</div>
-						<div class="perk-desc">
-							Get 10% off any drink at Starbucks with blah blah blah.
+		<div class="section side-by-side">
+			<div class="section card">
+				<div class="section title">
+					<span>Hot</span>
+				</div>
+				<div class="section perk-list" id="top-perks">
+					<div class="loader"></div>
+					<!--
+					<div class="section perk-tile">
```

Comment:
I'm guessing that you probably meant to delete this temporary stuff.

Reply:
No, it's there for reference since those divs aren't added until runtime. It can be removed when we "launch" the product lol.

Reply:
Oh that makes sense.